toolset/databases/mongodb/create.js

```diff
@@ -7,6 +7,14 @@ for (var i = 1; i <= 10000; i++) {
 db.world.createIndex({_id: 1})
 db.world.createIndex({id: 1})
 
+db.cachedworld.drop()
+for (var i = 1; i <= 10000; i++) {
+  db.cachedworld.save( { _id: i, id: i, randomNumber: (Math.floor(Math.random() * 10000) + 1) })
```

Comment:
There should not be a cachedworld table in any of the databases.  Please undo the changes to mongodb/create.js, mysql/create.sql, and postgres/create-postgres.sql.

Reply:
Please read the docs at http://frameworkbenchmarks.readthedocs.io/en/latest/Project-Information/Framework-Tests/#caching, 
`The request handler must retrieve a set of CachedWorld objects, equal in count to the count parameter, from the CachedWorld database table. Note that the CachedWorld database table uses a schema idential to the World table used for Test #3 but is distinct to avoid confusion.`
Hence I added these new tables to all the schemas

Reply:
Oh interesting!  Our fault - that part of the docs is wrong/outdated.  We've decided these tests should use the same World table as the other tests.  There are already implementations of the cached queries test that do this.

We'll update the docs.  Sorry for the confusion.